src/NzbDrone.Core/Download/Clients/Porla/Porla.cs

```diff
@@ -0,0 +1,239 @@
+                RemovesCompletedDownloads = false, // should be determined through Flags From  somewhere? I think session?
+            };
+
+            // the default directory value is stored in the presets values... i think?
+            // but will not always be active if not set
+
+            // destDir = new OsPath(preset.GetValueOrDefault("save_path") as string);
+            // if (destDir.IsNotNullOrEmpty)
+            // {
+            //    status.OutputRootFolders = new List<OsPath> { _remotePathMappingService.RemapRemoteToLocal(Settings.Host, destDir) };
+            // }
+
+            return status;
+        }
+
+        private static IList<string> ConvertRemoteEpisodeToTags(RemoteEpisode remoteEpisode)
+        {
+            var tags = new List<string>
+            {
+                $"starr.series={remoteEpisode.Series.CleanTitle}",
+                $"starr.season={remoteEpisode.MappedSeasonNumber}",
+                $"starr.tvdbid={remoteEpisode.Series.TvdbId}"
+            };
+            return tags;
+        }
+
+        protected override string AddFromMagnetLink(RemoteEpisode remoteEpisode, string hash, string magnetLink)
+        {
+            var tags = Settings.SeriesTag ? ConvertRemoteEpisodeToTags(remoteEpisode) : null;
+            var torrent = _proxy.AddMagnetTorrent(Settings, magnetLink, tags);
+            return torrent.InfoHash.Hash;
+        }
+
+        protected override string AddFromTorrentFile(RemoteEpisode remoteEpisode, string hash, string filename, byte[] fileContent)
+        {
+            var tags = Settings.SeriesTag ? ConvertRemoteEpisodeToTags(remoteEpisode) : null;
+            var torrent = _proxy.AddTorrentFile(Settings, fileContent, tags);
+            return torrent.InfoHash.Hash;
+        }
+
+        protected override void Test(List<ValidationFailure> failures)
+        {
+            failures.AddIfNotNull(TestConnection());
+            if (failures.HasErrors())
+            {
+                return;
+            }
+
+            failures.AddIfNotNull(TestGetTorrents());
+        }
+
+        private ValidationFailure TestConnection()
+        {
+            try
+            {
+                var sysVers = _proxy.GetSysVersion(Settings);
+                var version = new Version(sysVers.Porla.Version);
+
+                if (version < new Version("0.37.0"))
+                {
+                    // return new ValidationFailure(string.Empty, _localizationService.GetLocalizedString("DownloadClientValidationErrorVersion",
+                    //        new Dictionary<string, object> { { "clientName", Name }, { "requiredVersion", "0.37.0" }, { "reportedVersion", version } }));
+                    _logger.Warn($"Porla: Unsure if your version is forwards compatible: {version}");
+                }
+
+                if (version > new Version("0.37.0"))
+                {
+                    _logger.Warn($"Porla: Unsure if your version is backwards compatible: {version}");
```

Comment:
If it's under active development then I don't think we should be super strict on the version.

Reply:
I am currently not throwing any exception, regardless of the version porla tells us, but I am logging when the version doesn't match my expectation, so when something breaks I know by looking at the logs that the version is different.

I am running the live test on `latest` so I can catch new issues as and when the

Reply:
Oh the `< 1.0` check!

That is just a general warning in the logs for others.
The checks above it are doing the actual comparability stuff